scripts/pre_commit_linter.py

```diff
@@ -180,6 +182,16 @@
 _MESSAGE_TYPE_FAILED = 'FAILED'
 
 
+def _is_filename_excluded_for_bad_patterns_check(filename):
+    """Checks if file is excluded from the bad patterns check
+
+    Returns:
+        a boolean: true if excluded
+    """
+    return (any(filename.startswith(bad_pattern)
+                for bad_pattern in BAD_PATTERNS['\t']['excluded_dirs'])
```

Comment:
You're right, but that's not the tuple itself. 12 is the length of the string 'assets/i18n/'. Putting `print type(BAD_PATTERNS['\t']['excluded_dirs'])` gives `<type 'str'>`. I googled and came across [this stackoverflow answer](http://stackoverflow.com/questions/12876177/why-do-tuples-with-only-one-element-get-converted-to-strings). Adding a comma after the string should make it a tuple, should I do that?